pkgs/applications/virtualization/containerd/default.nix

```diff
@@ -27,14 +27,12 @@ buildGoModule rec {
 
   buildInputs = [ btrfs-progs ];
 
-  buildFlags = [ "VERSION=v${version}" "REVISION=${src.rev}" ];
-
   BUILDTAGS = lib.optionals (btrfs-progs == null) [ "no_btrfs" ];
 
   buildPhase = ''
     runHook preBuild
     patchShebangs .
```

Comment:
This should be more precise.

Reply:
Feel free to submit a PR fixing this in both of the `containerd` derivations.